docs/experimental/mmsinkhorn.rst

```diff
@@ -0,0 +1,15 @@
+ott.experimental.mmsinkhorn
```

Comment:
Since experimental is right now fairly small, would maybe merge the contents of this file with `docs/experimental/index.rst`.

Reply:
but then we would revert to this whenever adding something new? maybe simpler to leave room :)

Reply:
ok, if you don't mind, i'll leave it as it is 👍